src/elements/emby-slider/emby-slider.js

```diff
@@ -20,6 +20,27 @@ import '../emby-input/emby-input';
         }
     }
 
+    /**
+     * Returns normalized slider step.
+     *
+     * @param {HTMLInputElement} range slider itself
+     * @param {number|undefined} step step
+     * @returns {number} normalized slider step.
+     */
+    function normalizeSliderStep(range, step) {
+        if (step > 0) {
+            return step;
+        }
+
+        step = parseFloat(range.step);
+
+        if (step > 0) {
+            return step;
+        }
+
+        return 1;
```

Comment:
What do you think about this?

```suggestion
        return Math.max(step, 1);
```

Reply:
`parseFloat(range.step)` returns `NaN` for `'auto'` or an invalid value. `Math.max` returns `NaN` in this case.